src/components/Modal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="modal-backdrop">
+  <div class="modal-backdrop" @click="close">
     <div class="modal">
```

Comment:
You could add a stop propagation of the click on this div.

Reply:
Hum. Let me have a quick look.

Reply:
Rather than using stop I guess the best option will be using self instead. 

It'll make the code more readable, if we think this event should only be fired on the _target_ element, leaving children out it. 

Will update in a few minutes.

Good catch btw!

Reply:
Hi, @carlan . Sorry for the late. Your solution is working just like we expected. 

Thank you again! :smile: